app-utils/yaml-parser.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+const yaml = require('js-yaml');
+const path = require('path')
+
+const retrieveApiKey = () => {
```

Comment:
Actually good approach to learn how node works with filesystem, but bad way to work with sensitive data